### PR TITLE
chore(package): use `npm` instead of `yarn`

### PR DESCRIPTION
**Pull request by AugustinMauroy**
## Related Issue

#1

### CODE REVIEW

File: .slicemachine/assets/customtypes/home/mocks.json [hunk 1]
```diff
@@ -1,111 +1,302 @@
...
-  "linked_documents": [],
-  "lang": "en-us",
-  "alternate_languages": [],
-  "data": {
-    "title": "did",
-    "description": "said",
-    "hero_title": "tea",
-    "hero_description": "own",
-    "hero_actions": [
+  "uid": {
```
Comment (chrisgervang) [inline .slicemachine/assets/customtypes/home/mocks.json:2]: Can you please provide some more context for this change? It's not clear how it was generated, and the content doesn't seem related to Open Visualization.

**Comment by chrisgervang** [comment]
`yarn` is free and widely adopted as well, so it wouldn't prevent anyone from contributing. All of our other visualization projects use `yarn`, so for consistency I'd like to stick with it.

**Comment by AugustinMauroy** [comment]
If you thinks that better to stay with `yarn` let's go to keep it
